components/staticFeeds/sample-messages.json

```diff
@@ -118,6 +118,199 @@
       "confirmButton": {
         "label": "Got it!"
       }
+    },
+
+    {
+      "id": "demo-high-priority-no-group-no-date",
+      "title": "No group. No date. High priority.",
+      "titleShort": "No group. No date. High priority.",
+      "description": "No group. No date. High priority.",
+      "descriptionShort": "No group. No date. High priority.",
+      "messageType": "notification",
+      "goLiveDate": null,
+      "expireDate": null,
+      "featureImageUrl": null,
+      "priority": "high",
+      "audienceFilter": {
+      "groups": [],
+      "dataUrl": null,
+      "dataObject": null,
+      "dataArrayFilter": null
+      },
+      "actionButton": {
+      "label": "Go",
+      "url": "http://www.google.com"
+      },
+      "moreInfoButton": null,
+      "confirmButton": null
+    },
+    {
+      "id": "demo-low-priority-no-group-no-date",
```

Comment:
Maybe characterize this as `no-priority` rather than `low`, since what we're talking about is no value for `priority`. Hypothetically could introduce a `low` value for priority that isn't the default in some future. This isn't important, just a nit.